test/integration/taskRequests.test.js

```diff
@@ -0,0 +1,286 @@
+const chai = require("chai");
+const sinon = require("sinon");
+const { expect } = chai;
+const chaiHttp = require("chai-http");
+
+const app = require("../../server");
+const authService = require("../../services/authService");
+const tasksModel = require("../../models/tasks");
+const taskRequestsModel = require("../../models/taskRequests");
+const addUser = require("../utils/addUser");
+const cleanDb = require("../utils/cleanDb");
+const userData = require("../fixtures/user/user")();
+const taskData = require("../fixtures/tasks/tasks")();
+chai.use(chaiHttp);
+
+const config = require("config");
+const { TASK_REQUEST_STATUS } = require("../../constants/taskRequests");
+const cookieName = config.get("userToken.cookieName");
+
+let jwt;
+let taskId;
+
+const member = userData[9];
+const member2 = userData[10];
+const superUser = userData[4];
+const activeMember = userData[0];
+
+describe("Task Requests", function () {
+  let userId, userId2;
+  after(async function () {
+    await cleanDb();
+  });
+
+  afterEach(async function () {
+    sinon.restore();
+  });
+
+  describe("GET / - gets tasks requests", function () {
+    describe("When the user is super user", function () {
+      before(async function () {
```

Comment:
I believe here we don't require to call these functions like `verifyToken`, `generateAuthToken`, and `users-model`.

You can mock these functions, and get work done. 

In testing I believe we shouldn't be doing actual operations/db-transactions.

Reply:
For example mocking `authService`
```js
sinon.stub(authService, "generateAuthToken").returns("valid_token");
sinon.stub(authService, "verifyAuthToken").returns({ userId: superUserId });
```